apps/mobile/src/components/ButtonChip.tsx

```diff
@@ -109,14 +109,15 @@ export function ButtonChip({
         'flex h-6 items-center justify-center rounded-sm px-2 bg-black',
         chipContainerClassNames.containerClassName,
         {
+          'w-20': width === 'fixed-tight',
           'w-24': width === 'fixed',
           'w-auto': width === 'grow',
         }
       )}
       style={style}
     >
       <Typography
-        className={clsx('text-sm', chipContainerClassNames.textClassName)}
+        className={clsx('text-xs', chipContainerClassNames.textClassName)}
```

Comment:
confirmed with fraser that we can reduce the text size across the board for these chips, even if used outside of the notification screen